package.json

```diff
@@ -14,6 +14,7 @@
     "apollo-link-http": "^1.5.17",
     "graphql": "^15.3.0",
     "graphql-tag": "^2.10.4",
+    "gsap": "^3.10.3",
```

Comment:
@jumpybuns I think this may have snuck its way back in. Can we remove it, by chance?

Reply:
ah yes. I mustve deleted only on the previous branch